mckhannah1901/Co-op-Blog-Project
# Ath 1104 navigate to posts list after updating

### JIRA
[ATH-1104: Navigate to Posts List after Editing](https://wpengine.atlassian.net/secure/RapidBoard.jspa?rapidView=522&projectKey=ATH&modal=detail&selectedIssue=ATH-1104)

### What Has Been Done
The user is now redirected to the posts list after editing a post

### How to verify
1. Verify that a non logged in user can only view blog posts but can't edit them
2. Log in as a user
3. View the list of posts
4. Select the edit button on the post you'd like to edit
5. Fill in the form and publish
6. If successful, a flash message will tell you the post has been updated successfully and will redirect to the list of posts
